.github/workflows/desktop-server-api.apps.yml

```
@@ -29,59 +30,94 @@ jobs:
 
       - name: Install latest version of NPM
-        run: "sudo npm install -g npm@9"
+        run: 'sudo npm install -g npm@11.6.2'
 
-      - name: Install latest node-gyp package
-        run: "sudo npm install --quiet -g node-gyp@10.2.0"
+      - name: Install globally node-gyp, ts-node and nx packages
+        run: 'sudo npm install --quiet -g node-gyp@10.2.0 ts-node@10.9.2 nx@^22.5.2'
+
+      - name: Configure Registry
+        if: secrets.VERDACCIO_TOKEN != ''
```

actionlint (1.7.11)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)